ui/ember-cli-build.js

```diff
@@ -34,8 +34,9 @@ const appConfig = {
       return `${config.rootURL.replace(/\/$/, '')}${filePath}`;
     },
   },
-  babel: {
-    plugins: [['inline-json-import', {}]],
```

Comment:
do we need this? i couldn't tell

Reply:
It looks like it's for importing json files directly, which we do in `icons.js` component file. I think we should keep it for now, until we are no longer using custom icons.  https://www.npmjs.com/package/babel-plugin-inline-json-import

Reply:
ah, yep, that makes sense. fixed!

Reply:
talked about this in slack but sharing for posterity -- even with this plugin removed our single import of a `json` file directly in [Icon.js](https://github.com/hashicorp/vault/blob/aa5d9f7b4992ceb46cce1d0602786b125e4448f0/ui/lib/core/addon/components/icon.js#L8) is still working. we haven't figured out exactly _how_ it's still working but since it is we're leaving this change in here. one of our recent upgrades likely gave us this functionality.